jobs/ShareMethod.groovy

```diff
@@ -200,4 +200,47 @@ def downloadManifest(String url, String target){
     }
 }
 
+def generateManifestFromPR(String manifest_path){
+    def manifest = new net.sf.json.JSONObject()
+    String repository_url = "https://github.com/${ghprbGhRepository}.git"
```

Comment:
Thank you, please comment at the begining of this function.
``` create a manifest file as manifest_path which specifies the info of repo/branch under PR test, according to variable ghprbGhRepository ( build-in variable defined by Jenkins plugin GHPRB) ```
otherwise, user will feel very confusing about this function.